version.cfg

```diff
@@ -1 +1 @@
-0.18.0
+0.19.0
```

Comment:
Should this be 0.18.1? Looks like not a lot of changes so it could be a patch release.

Reply:
done!